docs/shipment.md

```diff
@@ -67,18 +68,24 @@ You can start a initial creation of a shipment, leading to transmission to a rep
 
 `POST /api/v1/shipment`
 
-This requires the following parameters and conditions:
+This **requires** the following parameters as [form-encoded data](http://docs.python-requests.org/en/master/user/quickstart/#more-complicated-post-requests):
```

Comment:
dont link to python package.
it just requires json...

Reply:
But can you tell me it it is `multipart/form-data` or `application/x-www-form-urlencoded`

See https://stackoverflow.com/a/4073451/261210

We also must update this for all other endpoints, see https://github.com/o2r-project/o2r-web-api/issues/51